test/functional/rpc_signrawtransaction.py

```diff
@@ -168,6 +170,44 @@ def witness_script_test(self):
         assert 'complete' in spending_tx_signed
         assert_equal(spending_tx_signed['complete'], True)
 
+        # Now try with a P2PKH script as the witnessScript
+        embedded_addr_info = self.nodes[1].getaddressinfo(self.nodes[1].getnewaddress('', 'legacy'))
+        embedded_privkey = self.nodes[1].dumpprivkey(embedded_addr_info['address'])
+        witness_script = embedded_addr_info['scriptPubKey']
+        redeem_script = hexlify(CScript([OP_0, sha256(check_script(witness_script))])).decode()
```

Comment:
nit. I think this might work:

```suggestion
        redeem_script = CScript([OP_0, sha256(check_script(witness_script))]).hex()
```

Reply:
Done